TESTS/write_fixtures.py

```diff
@@ -0,0 +1,58 @@
+# To regression test our wrappers we need examples. This script generates some example files, and re-generates them for testing. We can then check that the files are identical
+                       
+def Recreate_correlation_matrix_fixture(folder):
+    import os
+    ##### generate the correlation matrix using the data in EXAMPLE_DATA #####  
+    corrmat_path = os.getcwd()+folder+'/corrmat_file.txt'
+    from corrmat_from_regionalmeasures import corrmat_from_regionalmeasures
+    corrmat_from_regionalmeasures("EXAMPLE_DATA/PARC_500aparc_thickness_behavmerge.csv", "EXAMPLE_DATA/500.names.txt", corrmat_path, names_308_style=True)
+     
+def Recreate_network_analysis_fixture(folder, corrmat_path):
```

Comment:
Mega nitpicky request - can we rename `Recreate_network_analysis_fixture` and `Recreate_correlation_matrix_figure` to have lowercase rs at the beginning?